gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/RestApiController.groovy

```diff
@@ -69,8 +70,12 @@ trait RestApiController implements RequestJsonSupport, RestResponder, RestRegist
     }
 
     void handleException(Exception e) {
-        Problem apiError = problemHandler.handleException(e)
-        respondWith(apiError)
+        //Do nothing if its "broken pipe" exception
+        if(isBrokenPipe(e)) return
```

Comment:
is this here only for the jasper reports? do we really need the check just for that?

Reply:
It could happen for any action, i recently show it in logs for a regular "list" action  @basejump